react/Pill/Pill.js

```diff
@@ -27,25 +27,29 @@ export default class Pill extends Component<Props> {
   }
 
   renderStaticPill() {
-    const { text, className, ...restProps } = this.props;
+    const { children, text, className, ...restProps } = this.props;
+    const content = children || text;
+
     return (
       <span className={classnames(className, styles.staticPill)} {...restProps}>
-        <Text baseline={false} raw>{text}</Text>
+        <Text baseline={false} raw>{content}</Text>
```

Comment:
Sorry if i missed this, but should `children` be rendered inside `<Text>` or instead of?

Reply:
Inside, since we don't want arbitrary text styles within a pill.

Reply:
Since we're turning `baseline` off that should be fine